packs/windows-iad-sami-compliance.conf

```diff
@@ -0,0 +1,219 @@
+      "platform": "windows",
+      "description" : "Check ASLR configuration"
+    },
+    "KernelSehopEnabled": {
+      "query" : "select * from registry where key='HKEY_LOCAL_MACHINE\\SYSTEM\\CurrentControlSet\\Control\\Session Manager\\Kernel\\KernelSEHOPEnabled'",
+      "interval": "86400",
+      "snapshot": true,
+      "platform": "windows",
+      "description" : "Wether SEHOP is enabled"
+    },
+    "enableCertPaddingCheck": {
+      "query" : "select * from registry where key='HKEY_LOCAL_MACHINE\\SOFTWARE\\Microsoft\\Cryptography\\WinTrust\\Config\\EnableCertPaddingCheck'",
+      "interval": "86400",
+      "snapshot": true,
+      "platform": "windows"
+    },
+    "enableCertPaddingCheck_wow64": {
+      "query" : "select * from registry where key='HKEY_LOCAL_MACHINE\\SOFTWARE\\Wow6432Node\\Microsoft\\Cryptography\\WinTrust\\Config\\EnableCertPaddingCheck'",
+      "interval": "86400",
+      "snapshot": true,
+      "platform": "windows"
+    },
+    "cwdIllegalInDllSearch": {
+      "query" : "select * from registry where key='HKEY_LOCAL_MACHINE\\SYSTEM\\CurrentControlSet\\Control\\Session Manager\\CWDIllegalInDllSearch'",
+      "interval": "86400",
+      "snapshot": true,
+      "platform": "windows"
+    },
+    "disabledExceptionChainValidation": {
+      "query" : "select * from registry where key='HKEY_LOCAL_MACHINE\\SYSTEM\\CurrentControlSet\\Control\\Session Manager\\kernel\\DisableExceptionChainValidation'",
+      "interval": "86400",
+      "snapshot": true,
+      "platform": "windows"
+    },
+    "enableLowVaAccess": {
+      "query" : "select * from registry where key='HKEY_LOCAL_MACHINE\\SYSTEM\\CurrentControlSet\\Control\\Session Manager\\Memory Management\\EnableLowVaAccess'",
+      "interval": "86400",
+      "snapshot": true,
+      "platform": "windows"
+    },
+    "cfg": {
+      "query" : "select * from registry where key='HKEY_LOCAL_MACHINE\\SYSTEM\\CurrentControlSet\\Control\\Session Manager\\Memory Management\\EnableCfg'",
+      "interval": "86400",
+      "snapshot": true,
+      "platform": "windows"
+    },
+    "app_executeOptions": {
+      "query" : "select * from registry where key like 'HKEY_LOCAL_MACHINE\\SOFTWARE\\%Microsoft\\Windows NT\\CurrentVersion\\Image File Execution Options\\%\\executeOptions'",
+      "interval": "86400",
+      "snapshot": true,
+      "platform": "windows"
+    },
+    "app_mitigationOptions": {
+      "query" : "select * from registry where key like 'HKEY_LOCAL_MACHINE\\SOFTWARE\\%Microsoft\\Windows NT\\CurrentVersion\\Image File Execution Options\\%\\MitigationOptions'",
+      "interval": "86400",
+      "snapshot": true,
+      "platform": "windows"
+    },
+    "appCompat": {
+      "query" : "select * from registry where key='HKEY_LOCAL_MACHINE\\SOFTWARE\\%Microsoft\\Windows NT\\CurrentVersion\\AppCompatFlags\\Layers'",
+      "interval": "86400",
+      "snapshot": true,
+      "platform": "windows"
+    },
+    "app_disabledExceptionChainValidation": {
```

Comment:
Similarly here, can you use consistent casing? either stick to upper CamlCase, lower camlCase, or snake_case?